styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,74 @@
 .footer{
-  
-}
\ No newline at end of file
+  .accordionTitle h6{
```

Comment:
It would be more modular to rename this class to `.footer-accordion-title` and drop the `h6`. That way we can apply that class directly to whatever the heading/text element is without breaking anything if we need/decide to change it. That class name will also pull it out of being nested in `.footer` making the selector more efficient.